srcalc/src/test/java/gov/va/med/srcalc/domain/calculation/PopulatedDisplayGroupTest.java

```diff
@@ -36,15 +36,21 @@ public final void testToString()
                 SampleModels.genderVariable());
         final Patient patient = SampleCalculations.dummyPatient(1);
         patient.getHealthFactors().add(new HealthFactor(LocalDate.now(), "Dummy health factor"));
+        // Test a PopulatedDisplayGroup with only variables
         final PopulatedDisplayGroup group = new PopulatedDisplayGroup(variables, patient);
+        // Test a PopulatedDisplayGroup with variables and reference information
         final PopulatedDisplayGroup group2 = new PopulatedDisplayGroup(
                 Arrays.asList(SampleModels.functionalStatusVariable()), patient);
+        // Test a PopulatedDisplayGroup with only reference information
+        final PopulatedDisplayGroup group3 = new PopulatedDisplayGroup(
+                SampleModels.medicationsVariableGroup(), patient);
         assertEquals(
                 "Display Group 'Demographics' with display items [Age, Gender]",
                 group.toString());
         // Make sure the toString works with reference information too.
         assertEquals("Display Group 'Clinical Conditions or Diseases - Recent' with display items [Health Factors, Functional Status]",
                 group2.toString());
+        assertEquals("Display Group 'Medications' with display items [Active Medications]", group3.toString());
```

Comment:
I like the thoroughness, but it seems to me just one test for `toString()` is sufficient. Aren't you really testing the automatic addition of reference info here? In that case, testing the accessors would be a better test.